docs/lib/analytics/fragments/android/getting-started.md

```diff
@@ -1,4 +1,4 @@
-When your backend is successfully updated, there should be two newly created files: `amplifyconfiguration.json` and `awsconfiguration.json` in your project folder.
+When your backend is successfully updated, there should be one newly created file: `amplifyconfiguration.json` in your project folder.
```

Comment:
What value is this line providing?

Reply:
So, I noticed in storage docs also has this line. If you think it's unnecessary, I can delete it